src/Microsoft.Health.CosmosDb/Features/Health/ResponseWriter.cs

```diff
@@ -0,0 +1,44 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Net.Mime;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+using Newtonsoft.Json;
+
+namespace Microsoft.Health.CosmosDb.Features.Health
+{
+    public static class ResponseWriter
+    {
+        public static async Task HealthResponseWriter(HttpContext httpContext, HealthReport healthReport)
```

Comment:
Perhaps just call this method `WriteJson` which is in line with the built-in (HealthCheckResponseWriters.WriteMinimalPlaintext)